docs/getting_started/tutorial.rst

```diff
@@ -4,64 +4,121 @@ Your first Wagtail site
+
+.. important::
+   Before installing Wagtail, it is necessary to install the **libjpeg** and **zlib** libraries, which provide support for working with JPEG, PNG and GIF images (via the Python **Pillow** library).
+   The way to do this varies by platform—see Pillow's `platform-specific installation instructions <http://pillow.readthedocs.org/en/latest/installation.html#external-libraries>`_.
 
-       $ pip install wagtail
 
-2. Start your site:
+Create and activate a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. code-block:: console
+We recommend using a virtual environment, which provides an isolated Python environment. This tutorial uses `venv <https://docs.python.org/3/tutorial/venv.html>`_, which is packaged with Python 3.
 
-       $ wagtail start mysite
-       $ cd mysite
+Create a virtual environment:
 
-   Wagtail provides a ``start`` command similar to
-   ``django-admin.py startproject``. Running ``wagtail start mysite`` in
-   your project will generate a new ``mysite`` folder with a few
-   Wagtail-specific extras, including the required project settings, a
-   "home" app with a blank ``HomePage`` model and basic templates and a sample
-   "search" app.
+.. code-block:: console
 
-3. Install project dependencies:
+   $ python3 -m venv mysite-env
 
-   .. code-block:: console
 
-       $ pip install -r requirements.txt
+Activate your virtual environment:
 
-   This ensures that you have the relevant version of Django for the project you've just created.
+   On Windows:
 
-4. Create the database:
+      .. code-block:: console
 
-   .. code-block:: console
+         $ mysite-env\Scripts\activate.bat
 
-       $ python manage.py migrate
+   On Unix or MacOS (bash):
 
-   If you haven't updated the project settings, this will be a SQLite
-   database file in the project directory.
+      .. code-block:: console
 
-5. Create an admin user:
+         $ source mysite-env/bin/activate
+
+   For other shells, see the `venv documentation <https://docs.python.org/3/library/venv.html>`_.
+
+Install Wagtail
+~~~~~~~~~~~~~~~
+
+Use pip, which is packaged with Python, to install Wagtail and its dependencies:
+
+.. code-block:: console
+
+   $ pip install wagtail
+
+Generate your site
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+   $ wagtail start mysite
+   $ cd mysite
+
+Wagtail provides a ``start`` command similar to
+``django-admin.py startproject``. Running ``wagtail start mysite`` in
+your project will generate a new ``mysite`` folder with a few
+Wagtail-specific extras, including the required project settings, a
+"home" app with a blank ``HomePage`` model and basic templates and a sample
+"search" app.
+
+Install project dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+   $ pip install -r requirements.txt
+
+This ensures that you have the relevant version of Django and any other dependencies for the project you have just created.
```

Comment:
This text is changed slightly so that it's more about what this does generally. I'm not attached to the wording if anyone thinks it should be different.

Reply:
Most important is that this ensures the used Wagtail:

`This ensures that you have the relevant version of Wagtail, Django and any other dependencies for the project you have just created.`

Reply:
I did the following (only difference is I ended up saying "versions")
>This ensures that you have the relevant versions of Wagtail, Django, and any other dependencies for the project you have just created.

https://github.com/wagtail/wagtail/pull/5210/files#diff-c9ff385ec9bf22ea0114be2b2cfa6e33R90